src/PageCharacter/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -185,6 +185,10 @@ export default function TabBuild() {
     const finalizedList: Promise<FinalizeResult>[] = []
     for (let i = 0; i < maxWorkers; i++) {
       const worker = new Worker(new URL('./BackgroundWorker.ts', import.meta.url))
+      worker.addEventListener("error", e => {
```

Comment:
In theory, replicable worker errors will always refresh the page, making it hard to trace. 
should probably lock this behind `process.env.NODE_ENV === "production"`.